setup.py

```diff
@@ -32,11 +32,10 @@
         "Programming Language :: Python :: 3.11",
         "Programming Language :: Python :: 3.12",
         "Programming Language :: Python :: 3.13",
-        "Programming Language :: Python :: 3.9",
```

Comment:
could potentially add 3.14 here, as it _should_ [release tomorrow](https://peps.python.org/pep-0745/#schedule)

Reply:
Ahh well spotted. I think I've kept you both waiting long enough though so will merge this and test against 3.14 later on. 👍🏻